src/libpfc.c

```diff
@@ -1,4 +1,6 @@
 /* Includes */
+#define _GNU_SOURCE
```

Comment:
This seems to be the [right way](https://stackoverflow.com/a/7297011/149138) to do this and solved some unusual compile-errors with `c99` (although I ended up going with `gnu99`). 

In particular it seems you should use _GNU_SOURCE before any system header is included.